microsoftbotframework/conversationoperations.py

```diff
@@ -30,7 +30,7 @@ def send(self):
                                     "/v3/conversations/{}/activities".format(
                                         self.conversation['id']))
 
-        response = self._request(response_url, 'post', self.to_dict())
+        response = self._request(response_url, 'post', self.to_dict(), self.timeout_seconds)
```

Comment:
Please remove the timeout_seconds argument, it must be passed in at class instantiation per your other additions